librz/core/canalysis.c

```diff
@@ -378,9 +378,15 @@ static void bb_info_print(RzCore *core, RzAnalysisFunction *fcn, RzAnalysisBlock
 		inputs += (bb2->jump == bb->addr) + (bb2->fail == bb->addr);
 	}
 	if (bb->switch_op) {
-		RzList *unique_cases = rz_list_uniq(bb->switch_op->cases, casecmp);
-		outputs += rz_list_length(unique_cases);
-		rz_list_free(unique_cases);
+		RzPVector *unique_cases = rz_pvector_new(NULL);
+		void **it;
+		rz_pvector_foreach (bb->switch_op->cases, it) {
+			if (!rz_pvector_contains(unique_cases, *it)) {
```

Comment:
This is wrong. this contains only works for pointer comparison, not actual value comparison.

Reply:
you may have 2 different pointers with the same string.

Reply:
Will fix .